website/docs/wallet/withdraw-validator.md

```diff
@@ -190,7 +190,7 @@ By calling the command above, you should go through an interactive process that
 1. **Your mnemonic language**. You can see the different options available, where English is one of the options, among others
 2. **The network** you wish to perform this operation for. Select the `zhejiang` network for this tutorial
 3. Enter your **mnemonic** next
-4. Next, you will be asked for your the index of the withdrawal key you wish to use. For **most users**, this will be 0 for the first validator, 1 for their second, etc....
+4. Next, you will be asked for your starting index when you created your validators read more about hd wallets [here](https://eips.ethereum.org/EIPS/eip-2334#path). For **most users**, this will be 0 unless you created validators from a non default index.
```

Comment:
What's a `starting`/`non default` index? Are these well-known terms for the audience?

Reply:
All in all pretty confusing, if you use the staking deposit cli by default it will create keys starting with index 0, the path in the hd wallet is required to access the associated keys. if you used the same mnemonic to generate more keys you'd start from the last index and continue generating there. I'm not super familiar but I think this tries to address those situations. I asked if the staking deposit cli could have this optional and just start 0 by default.

Reply:
I updated this with additional info